apps/meteor/app/2fa/client/TOTPGoogle.js

```diff
@@ -25,8 +24,8 @@ const loginWithGoogleAndTOTP = function (options, code, callback) {
 	// has the proper email address after the OAuth conversation.
 
 	if (typeof Accounts._options.restrictCreationByEmailDomain === 'string') {
-		options = _.extend({}, options || {});
-		options.loginUrlParameters = _.extend({}, options.loginUrlParameters || {});
+		options = Object.assign({}, options || {});
```

Comment:
Do we need the assign? Can we use spread? 👀

Reply:
Maybe. I didn't like to take the risk because object spread and `Object.assign()` work different when there are property setters involved.